### ISSUE TITLE
Warnings from WP CountUp JS's CountUp block in blocks.wp-a2z.org

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/bobbingwide#60]
In blocks.wp-a2z.org I documented the CountUp block without seeing any Warnings.
But in the version cloned to https://blocks.wp-a2z.org/block/countup-roelmagdaleno-wp-countup-js/
I got 4 warnings and there were a couple of other problems.

**Question:** Why didn't I see the warnings in my development environment?

**Comment by bobbingwide** [comment]
On blocks.wp.a2z 
```
<!-- wp:roelmagdaleno/wp-countup-js {"end":6636561134,"fontSize":32,"start":6506664745} -->
<div class="wp-block-roelmagdaleno-wp-countup-js"><div data-start="6506664745" data-duration="2" data-delay="0" data-decimal="." data-decimals="0" data-separator="," data-suffix="" data-prefix="" data-grouping="true" data-easing="true" data-scroll="false" data-reset="false" data-end="6636561134" class="counter" style="text-align: center; font-size: 32px;">6506664745</div></div>
<!-- /wp:roelmagdaleno/wp-countup-js -->
```


On blocks.wp-a2z.org

```
<!-- wp:roelmagdaleno/wp-countup-js {"end":6636561134,"fontSize":32,"start":6506664745} -->
<div class="wp-block-roelmagdaleno-wp-countup-js"><div data-start="6506664745" data-duration="2" data-delay="0" data-decimal="." data-decimals="0" data-separator="," data-suffix="" data-prefix="" data-grouping="true" data-easing="true" data-scroll="false" data-reset="false" data-end="6636561134" class="counter" style="text-align: center; font-size: 32px;">6506664745</div></div>
<!-- /wp:roelmagdaleno/wp-countup-js -->
```

After fixup in the block editor.
```
<!-- wp:roelmagdaleno/wp-countup-js {"end":6636561134,"fontSize":32,"start":6506664745} -->
<div class="wp-block-roelmagdaleno-wp-countup-js"><div data-start="6506664745" data-duration="2" data-delay="0" data-decimal="null" data-decimals="0" data-separator="null" data-suffix="null" data-prefix="null" data-grouping="false" data-easing="false" data-scroll="false" data-reset="false" data-end="6636561134" class="counter" style="text-align: center; font-size: 32px;">6506664745</div></div>
<!-- /wp:roelmagdaleno/wp-countup-js -->
```

**Comment by bobbingwide** [comment]
> **Question:** Why didn't I see the warnings in my development environment?
**Answer:**  When  I installed the plugin in the development environment the batch routine activated the plugin in order for me to document the block(s).

In the live environment I didn't activate the plugin. I just ran oik-loader after cloning. Consequently the plugin's options weren't initialised. This led to the error messages.

Problem resolution: Activate the plugin in the live site.
